File: problem171.cs
public class Solution {
    public int TitleToNumber(string s) {
        // To convert letter to integer number: char.ToUpper(c) - 64
        int col = 0;
        int multiplier = 1;
        for (int i = s.Length - 1; i >= 0; i--) {
            col += multiplier * (char.ToUpper(s[i]) - 64);
            multiplier *= 26;
        }
        return col;
    }
}

File: su19/problem171.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        col = 0
        multiplier = 1
        i = len(s) - 1
        while i >= 0:
            rank = ord(s[i].lower()) - 96
            col += rank * multiplier
            multiplier *= 26
            i -= 1
        return col